src/main/java/core/basesyntax/ShopServiceAndReportGenerator/ShopServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.ShopServiceAndReportGenerator;
+
```

Comment:
There is a typo in the package name 'OpationStrategy'. It should likely be 'OperationStrategy'. Please correct the package name in the import statement to ensure proper functionality.